### PR TITLE
LPS-168551 HomeExtPropertiesReturnsError testcase

### PR DESCRIPTION
**Pull request by sofiamendozalr**

### CODE REVIEW

**Comment by sofiamendozalr** [comment]
ci:test:sf

**Comment by lucasperj** [comment]
ci:test:sf

**Comment by sofiamendozalr** [comment]
ci:test:upgrade:no-compile

**Comment by sofiamendozalr** [comment]
ci:test:sf

**Comment by lucasperj** [comment]
ci:test:upgrade:no-compile

**Comment by lucasperj** [comment]
ci:test:upgrade:nocompile

**Comment by lucasperj** [comment]
ci:test:sf

File: modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml
Comment (susanchen3) [inline modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml]: I think we should modify the substring to "portal-ext.properties prior to running an upgrade to prevent possible conflicts." because CI's home user is different

Comment (susanchen3) [reply modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml]: Or "Remove /${USER}/portal-ext.properties prior to running an upgrade to prevent possible conflicts." would be better if you can find the user

**Review by susanchen3 — CHANGES_REQUESTED** [review]
Currently the [test](https://testray.liferay.com/home/-/testray/case_results/2071216177) is failing because it is not matching the substring correctly

File: modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml
Comment (lucasperj) [reply modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml]: She fixes that at : https://github.com/lucasperj/liferay-portal/pull/311/commits/c2c8153fa0fd78c93edae6a0ab82da2547254658

Comment (lucasperj) [reply modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml]: `/home/${user.name}/portal-ext.properties` can work?

Comment (susanchen3) [reply modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml]: o woops didnt see that

Comment (susanchen3) [reply modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml]: ignore these requested changes

File: modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml
Comment (susanchen3) [inline modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml:325]: Change substring to "Remove ${user.home}/portal-ext.properties prior to running an upgrade to prevent possible conflicts."

File: modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml
Comment (susanchen3) [inline modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml:330]: Change message to "Upgrade client is attempting to start an upgrade process with portal-ext.properties in user's home directory."

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/DatabaseUpgradeClient.testcase
Comment (susanchen3) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/DatabaseUpgradeClient.testcase:67]: Rename to check-upgrade-client-home-ext-properties

File: modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml
Comment (susanchen3) [inline modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml:302]: Rename to check-upgrade-client-home-ext-properties

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/DatabaseUpgradeClient.testcase
Comment (susanchen3) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/upgrades/DatabaseUpgradeClient.testcase:62]: I think we can simplify the description. Maybe to something like: "LPS-168551: Upgrade client throws error when portal-ext.properties exists in home directory"

File: modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml
Comment (susanchen3) [inline modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml:333]: I think we should remove the portal-ext.properties file from the home directory when we clean up the test too :)

**Review by susanchen3 — CHANGES_REQUESTED** [review]
Great work @sofiamendozalr! Just a few suggestions and fixes :D

File: modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml
Comment (susanchen3) [reply modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml:325]: Looks like [CI's output](https://testray.liferay.com/reports/production/logs/2022-11/test-1-30/test-portal-acceptance-pullrequest(master)/5470/functional-upgrade-tomcat90-mysql57-jdk8/0/7/jenkins-console.txt.gz?authuser=0) is "Remove /root/portal-ext.properties prior to running an upgrade to prevent possible conflicts."

On CI, it seems to just be _/root_ instead of _/home/root_, so I think `${user.home}` is better than `${user.name}`. 

`${user.home}` covers _/home/user/_ for me locally.

**Comment by sofiamendozalr** [comment]
ci:test:sf

**Comment by sofiamendozalr** [comment]
ci:test:upgrade:nocompile

File: modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml
Comment (sofiamendozalr) [reply modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml:333]: thank you for everything, I fixed all :)

**Comment by lucasperj** [comment]
ci:test:sf

**Comment by lucasperj** [comment]
ci:test:upgrade:nocompile

**Comment by susanchen3** [comment]
results are importing :hourglass_flowing_sand:  : https://test-1-38.liferay.com/view/All/job/publish-testray-report/11110/

**Comment by susanchen3** [comment]
Nice, test seems to be passing now: https://testray.liferay.com/home/-/testray/case_results/2073259921

Although, we might want to add `@disable-webdriver = "true"` to the test because it's not using the UI to save on resources

**Comment by sofiamendozalr** [comment]
ci:test:sf

**Comment by sofiamendozalr** [comment]
ci:test:sf

**Comment by sofiamendozalr** [comment]
ci:test:upgrade:nocompile

**Comment by sofiamendozalr** [comment]
ci:test:upgrade:nocompile

**Comment by sofiamendozalr** [comment]
ci:test:upgrade

**Comment by sofiamendozalr** [comment]
ci:test:sf

**Comment by sofiamendozalr** [comment]
ci:test:upgrade

**Comment by lucasperj** [comment]
Same, to me too! Very good Sofia :smiley:

**Comment by lucasperj** [comment]
Can you please pull and push your master, rebase your branch after that and execute : `git push origin <branch_id> -f` to have your branch closed in your local too, then you can open a new PR to @vicnate5. 
Update the description of the new pr adding the issue link : {ticket JIRA link} and the original PR link (this one), [example](https://github.com/vicnate5/liferay-portal/pull/2198). 
Finally lets wait for him to do the final review and forward that :rocket: , thanks a lot for the work on that Sofia, congratulations, and thanks @susanchen3 for the help to understand the UpgradeClient tests.

**Comment by lucasperj** [comment]
ci:close

**Pull request closed without merging**
